### PR TITLE
Migration of an existing Ada/SPARK project to Alire

### PR DESCRIPTION
**Pull request by Fabien-Chouteau**
@pat-rogers if you have a minute to review this :+1:

### CODE REVIEW

File: doc/getting-started.md [hunk 1]
```diff
@@ -146,3 +146,83 @@ of the `testsuite` folder.
...
+Clone your project repository and enter the directory:
+```bash
+$ git clone https://github.com/github_login/my_crate.git
+$ cd my_crate
+```
+
+At this point you have a choice:
+ 1. Let Alire generate a new GPR project file for you. This is recommended for most projects, and in particular if your project has simple code organization and GPR project file. One of the advantages is that Alire will create a GPR project file “standardized” for best integration in the ecosystem.
+
+ 1. Keep your existing GPR project file. This is recommended for projects with complex GPR project file(s).
```
Comment (pat-rogers) [inline doc/getting-started.md:170]: 2

**Review by pat-rogers — CHANGES_REQUESTED** [review]

File: doc/getting-started.md (hunk 1 above)
Comment (mosteo) [reply doc/getting-started.md:170]: Pat, this is okay, as the Markdown viewer will properly number enumerated lists.

File: doc/getting-started.md [hunk 2]
```diff
@@ -146,3 +146,83 @@ of the `testsuite` folder.
...
+`project-files` field in your `alire.toml` manifest. For instance:
+```toml
+project-files = ["project_file.gpr"]
+```
+Although this is not recommended (see Best practices), you can have multiple GPR project files:
+```toml
+project-files = ["project_file_1.gpr", "project_file_2.gpr"]
+```
+
+You can now to compile your crate with:
```
Comment (mosteo) [inline doc/getting-started.md]: Remove "to"

File: doc/getting-started.md (hunk 1 above)
Comment (pat-rogers) [reply doc/getting-started.md:170]: OK, thanks.

**Review by mosteo — APPROVED** [review]

**Pull request merged**